uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/contract-entitlements.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+  'common/moment',
+  'leave-absences/absence-tab/modules/components',
+  'common/models/contract'
+], function (_, moment, components) {
+  components.component('contractEntitlements', {
+    bindings: {
+      contactId: '<'
+    },
+    templateUrl: ['settings', function (settings) {
+      return settings.pathTpl + 'components/contract-entitlements.html';
+    }],
+    controllerAs: 'entitlements',
+    controller: ['$log', '$q', 'HR_settings', 'AbsenceType', 'Contract', controller]
+  });
+
+  function controller ($log, $q, HRSettings, AbsenceType, Contract) {
+    $log.debug('Component: contract-entitlements');
+
+    var vm = {};
+
+    vm.contactId = this.contactId;
+    vm.contracts = [];
+    vm.contractsLoaded = false;
+    vm.absenceTypes = [];
+
+    (function init () {
+      return $q.all([
+        loadAbsenceTypes()
+      ])
+      .then(function () {
+        return loadContracts();
+      });
+    })();
+
+    /**
+     * Formats the date according to user settings
+     *
+     * @param {object} date
+     */
+    function formatDate (date) {
+      var userDateFormat = HRSettings.DATE_FORMAT;
+      var dateFormat = userDateFormat ? userDateFormat.toUpperCase() : 'DD/MM/YYYY';
```

Comment:
But we need to know why HR_settings is null, I tried to find out
https://github.com/civicrm/civihr/blob/2afc263d0ff5b8a7a77c9f22df621ccaa873e8f3/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/modules/config.js#L27-L27
This is how Date_Format is set. The same needs to be done in absence module, preferably in app.js. Then the condition check can be removed from here.